vulture/utils.py

```diff
@@ -0,0 +1,54 @@
+import codecs
+import os
+import re
+import tokenize
+
+# Encoding to use when converting input files to unicode.
+ENCODING = 'utf-8'
+
+# The ast module in Python 2 trips over "coding" cookies, so strip them.
+ENCODING_REGEX = re.compile(
+    r"^[ \t\v]*#.*?coding[:=][ \t]*([-_.a-zA-Z0-9]+).*?$", flags=re.M)
```

Comment:
I'd rather keep ENCODING_REGEX in core.py since it'll only be used in the scan() function.